djangocms_versioning/monkeypatch/extensions.py

```diff
@@ -0,0 +1,26 @@
+from cms.extensions.extension_pool import ExtensionPool
+from cms.models.titlemodels import PageContent
+
+
+def _copy_title_extensions(self, source_page, target_page, language, clone=False):
```

Comment:
One thing thta we have never done which is bad is add a comment referring to the commit / code state that the code is copied from. That would make upgrading it, or even changing it in the future much easier. you could guess from the date of the PR but being explicit is always better. Something like: 

# Taken from djangocms-cms/extensions/admin.py, last changed in: https://github.com/divio/django-cms/commit/2894ae8bcf92092d947a097499c01ab2bbb0e6df

Strangely my commit.